docs/the_black_code_style/future_style.md

```diff
@@ -0,0 +1,35 @@
+# The (future of the) Black code style
+
+```{warning}
+Changes to this document often aren't tied and don't relate to releases of
+_Black_. It's recommended that you read the latest version available.
+```
+
+## Using backslashes for with statements
+
+[Backslashes are bad and should be never be used](labels/why-no-backslashes) however
+there is one exception: `with` statements using multiple context managers. Before Python
+3.9 Python's grammar does not allow organizing parentheses around the series of context
+managers.
+
+We don't want formatting like:
+
+```py3
+with make_context_manager1() as cm1, make_context_manager2() as cm2, make_context_manager3() as cm3, make_context_manager4() as cm4:
+    ...  # nothing to split on - line too long
+```
+
+So _Black_ will eventually format it like this:
+
+```py3
+with \
+     make_context_manager(1) as cm1, \
+     make_context_manager(2) as cm2, \
+     make_context_manager(3) as cm3, \
+     make_context_manager(4) as cm4 \
+:
+    ...  # backslashes and an ugly stranded colon
+```
```

Comment:
I don't know if this is still true - I get code like this these days: https://github.com/cooperlees/json2netns/blob/main/src/json2netns/tests/netns.py#L43-L45

I guess maybe cause I don't have the '\'

Reply:
I'm confused, the style described in this document is planned, but hasn't be implemented yet. This used to exist the main style doc but that was misleading and confusing.